js/src/BarsModel.js

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 
-var d3 = Object.assign({}, require("d3-array"), require("d3-scale"));
+var d3 = Object.assign({}, require("d3-array"), require("d3-scale"), require("d3-scale-chromatic"));
```

Comment:
I can't pin down when it happened, but since v3.x I think d3.schemeCategory10 was split into d3-scale-chromatic.